src/PromiseState.js

```diff
@@ -11,9 +11,17 @@ export default class PromiseState {
   // creates as PromiseState that is refreshing
   // can be called without a previous PromiseState and will be both pending and refreshing
   static refresh(previous, meta) {
```

Comment:
I didn't address it here, but was noticing how it's a little weird that `meta` here is only used if `previous` is undefined; although, it does act like `resolve(value, meta)` though. I believe I did that this to simplify some of the logic on the refetch size, but it makes this API a little ambiguous. Don't think it needs to be addressed now, but just calling it out for the future.